packages/server/src/adapters/aws-lambda/utils.ts

```diff
@@ -162,3 +157,14 @@ export type APIGatewayPayloadFormatVersion =
 export const UNKNOWN_PAYLOAD_FORMAT_VERSION_ERROR_MESSAGE =
   'Custom payload format version not handled by this adapter. Please use either 1.0 or 2.0. More information here' +
   'https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-develop-integrations-lambda.html';
+
+export function lambdaEventToHTTPBody(event: APIGatewayEvent) {
+  let body: string | null | undefined;
+  if (event.body && event.isBase64Encoded) {
+    body = Buffer.from(event.body, 'base64').toString('utf8');
```

Comment:
Arent these gzipped too? I know cloudwatch events are sent as gzipped base64 (dont ask me how i have that in fresh memory 💀💀)

Reply:
This isn't new code, just moved around so already in the wild :)

API Gateway I believe should handle the gzipping generally, though base64 will be used because lambdas are weird